packages/esm-patient-test-results-app/src/trendline/trendline.component.tsx

```diff
@@ -85,6 +85,17 @@ const Trendline: React.FC<{
     [setRange, upperRange],
   );
 
+  /**
+   * reorder svg element to bring line in front of the area
+   */
+  React.useEffect(() => {
+    if (patientData)
+      setTimeout(() => {
```

Comment:
Why is this deferred? Would a `useLayoutEffect` be better here?

Also don't forget that `querySelector` may return `null`. Maybe use `?.`? It could be that future versions of Carbon implement that differently...

Reply:
`useLayoutEffect` was not on my radar – you're totally right, this is the right tool for the job.